proposal/prefixed-parent-selector.md

```diff
@@ -0,0 +1,58 @@
+# Prefixed Parent Selector: Draft 2
```

Comment:
```suggestion
# Prefixed Parent Selector: Draft 1
```

We only update draft numbers when the *committed* version of the proposal changes; there's no need to do so for pull request review changes.